### PR TITLE
Added definitions for some upcoming Pimoroni LED driver boards

### PR DESCRIPTION
**Pull request by ZodiusInfuser**

### CODE REVIEW

**Comment by ZodiusInfuser** [comment]
This PR is ready for review. Tested both configs on the intended hardware.
PRs for CircuitPython.org will follow

File: ports/raspberrypi/boards/pimoroni_interstate75/mpconfigboard.h [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+#define MICROPY_HW_ROW_A (&pin_GPIO6)
+#define MICROPY_HW_ROW_B (&pin_GPIO7)
+#define MICROPY_HW_ROW_C (&pin_GPIO8)
+#define MICROPY_HW_ROW_D (&pin_GPIO9)
+#define MICROPY_HW_ROW_E (&pin_GPIO10)
+
+#define MICROPY_HW_CLK (&pin_GPIO11)
+#define MICROPY_HW_LAT (&pin_GPIO12)
+#define MICROPY_HW_OE (&pin_GPIO13)
```
Comment (tannewt) [inline ports/raspberrypi/boards/pimoroni_interstate75/mpconfigboard.h:19]: What are these for? Is it an RGB matrix? I *think* you could init it in `board_init` to have it display the serial by default.

**Review by tannewt — CHANGES_REQUESTED** [review]
Look like some of the `#defines` for MicroPython have leaked into these board defs like `MICROPY_HW_USER_SW`. I think it'd be best to remove the ones that circuitpython doesn't use. That'll simplify the files and make them clearer.

File: ports/raspberrypi/boards/pimoroni_interstate75/mpconfigboard.h (hunk 1 above)
Comment (ZodiusInfuser) [reply ports/raspberrypi/boards/pimoroni_interstate75/mpconfigboard.h:19]: Yes. HUB75 style panels. Tested already with the rgbmatrix library.

"Have it display the serial by default." Guessing you mean the matrix serial, not UART? Perhaps, but would that confuse things for anyone following guides?

Comment (jepler) [reply ports/raspberrypi/boards/pimoroni_interstate75/mpconfigboard.h:19]: We haven't yet enabled an RGBMatrix as the built in display of a board, so there may be problems here.

Typically these panels are 64x32 at the greatest, leaving room for about 2 full rows of 6 characters or something; in my experience this hasn't been useful for viewing REPL errors.  And of course if it's just a header, not integrated with a specific size of matrix, the display might be jumbled depending what the board is plugged in to.

Comment (ZodiusInfuser) [reply ports/raspberrypi/boards/pimoroni_interstate75/mpconfigboard.h:19]: The board is just a header connection (like MatrixPortal) so the problems you describe would likely be the case here too.

**Comment by ZodiusInfuser** [comment]
From discussing with @tannewt, I've removed the excess `MICROPY` and `IGNORE_PIN` defines

**Review by tannewt — APPROVED** [review]
Much simpler! Thank you!

**Comment by ZodiusInfuser** [comment]
Thank you very much!

**Pull request merged**